documentation/src/components/component.ts

```diff
@@ -62,6 +65,27 @@ class ComponentElement extends LayoutElement {
         });
     }
 
+    loadDocs() {
```

Comment:
Lazily load the documentation. `ComponentApiDocs` will theoretically already be available, but having it here means when we move to the updated side nav process that we should get that off the initial load as well!